data-pipeline/csv-validation/petl-validation.py

```diff
@@ -0,0 +1,51 @@
r Setback (# of feet)', '2-Family Max. Lot Coverage - Buildings (%)', '2-Family Max. Lot Coverage - Buildings & Impervious Surface (%)', '2-Family Min. # Parking Spaces Per Studio or 1BR', '2-Family Min. # Parking Spaces Per 2+ BR', '2-Family Max. Height (# of stories)', '2-Family Max. Height (# of feet)', '2-Family Floor to Area Ratio', '2-Family Min. Unit Size (SF)', '3-Family Affordable Housing Only', '3-Family Elderly Housing Only', '3-Family Min. Lot (ACRES)', '3-Family Max. Density (UNITS/ACRE)', '3-Family Front Setback (# of feet)', '3-Family Side Setback (# of feet)', '3-Family Rear Setback (# of feet)', '3-Family Max. Lot Coverage - Buildings (%)', '3-Family Max. Lot Coverage - Buildings & Impervious Surface (%)', '3-Family Min. # Parking Spaces Per Studio or 1BR', '3-Family Min. # Parking Spaces Per 2+ BR', '3-Family Connection to Sewer and/or Water Required', '3-Family Connection or Proximity to Public Transit Required', '3-Family Max. Height (# of stories)', '3-Family Max. Height (# of feet)', '3-Family Floor to Area Ratio', '3-Family Min. Unit Size (SF)', '3-Family Max. # Bedrooms Per Unit', '4+-Family Affordable Housing Only', '4+-Family Elderly Housing Only', '4+-Family Min. Lot (ACRES)', '4+-Family Max. Density (UNITS/ACRE)', '4+-Family Front Setback (# of feet)', '4+-Family Side Setback (# of feet)', '4+-Family Rear Setback (# of feet)', '4+-Family Max. Lot Coverage - Buildings (%)', '4+-Family Max. Lot Coverage - Buildings & Impervious Surface (%)', '4+-Family Min. # Parking Spaces Per Studio or 1BR', '4+-Family Min. # Parking Spaces Per 2+ BR', '4+-Family Connection to Sewer and/or Water Required', '4+-Family Connection or Proximity to Public Transit Required', '4+-Family Max. Height (# of stories)', '4+-Family Max. Height (# of feet)', '4+-Family Floor to Area Ratio', '4+-Family Min. Unit Size (SF)', '4+-Family Max. # Bedrooms Per Unit', '4+-Family Max. # Units Per Building', 'Affordable Housing (AH) Treatment', 'AH - Definition', 'AH - Elderly Housing Only', 'AH Min. Lot (ACRES)', 'AH Max. Density (UNITS/ACRE)', 'AH Min. # Parking Spaces Per Studio or 1BR', 'AH Min. # Parking Spaces Per 2+ BR', 'AH Connection to Sewer and/or Water Required', 'AH Connection or Proximity to Public Transit Required', 'AH Min. Unit Size (SF)', 'AH Max. # Bedrooms Per Unit', 'AH Max. # Units Per  Building', 'Accessory Dwelling Unit (ADU) Treatment', 'ADU Employee or Family Occupancy Required', 'ADU Renter Occupancy Prohibited', 'ADU Owner Occupancy Required', 'ADU Elderly Housing Only', 'ADU Min. Lot (acres)', 'ADU Min. # Parking Spaces (Additional to Main Unit)', '"ADU Restricted to Only Primary Structure (i.e., No Outbuildings like Garages)"', 'ADU Max. Size (% of Main Unit)', 'ADU Max. Size (SF)', 'ADU Max. # Bedrooms Per Unit', 'Planned Residential Development (PRD) Treatment', 'Mobile or Manufactured Home Park (Y/N)', 'PRD Min. Lot (Acres)', 'PRD Max. Density (Units/Acre)', 'PRD Max. # Units Per Development', 'Special Notes', 'Tooltip Notes')
```

Comment:
Can you make this a multi-line tuple? If that's not possible, a multi-line list with a conversion to a tuple later on